Makefile

```diff
@@ -241,15 +241,25 @@ endif
 	@docker images --format "{{.CreatedAt}}\tID:{{.ID}}\t{{.Repository}}:{{.Tag}}" $(CTL_CONTAINER_NAME)
 	@docker images --format "{{.CreatedAt}}\tID:{{.ID}}\t{{.Repository}}:{{.Tag}}" $(CTL_CONTAINER_NAME):$(VERSION)
 
-	@echo "\nNow push the tag and images. Then create a release on Github and"
-	@echo "attach dist/calicoctl, dist/calicoctl-darwin-amd64, and dist/calicoctl-windows-amd64.exe binaries"
-	@echo "\nAdd release notes for calicoctl. Use this command"
-	@echo "to find commit messages for this release: git log --oneline <old_release_version>...$(VERSION)"
-	@echo "git push origin $(VERSION)"
-	@echo "docker push calico/ctl:$(VERSION)"
-	@echo "docker push quay.io/calico/ctl:$(VERSION)"
-	@echo "docker push calico/ctl:latest"
-	@echo "docker push quay.io/calico/ctl:latest"
+	@echo ""
+	@echo "# Push the created tag to GitHub"
+	@echo "  git push origin $(VERSION)"
+	@echo ""
+	@echo "# Now, create a GitHub release from the tag, add release notes, and attach the following binaries:"
+	@echo "- dist/calicoctl"
+	@echo "- dist/calicoctl-darwin-amd64"
+	@echo "- dist/calicoctl-windows-amd64.exe"
+	@echo "# To find commit messages for the release notes:  git log --oneline <old_release_version>...$(VERSION)"
+	@echo ""
+	@echo "# Now push the newly created release images."
+	@echo "  docker push calico/ctl:$(VERSION)"
+	@echo "  docker push quay.io/calico/ctl:$(VERSION)"
+	@echo ""
+	@echo "# For the final release only, push the latest tag"
+	@echo "# DO NOT PUSH THESE IMAGES FOR RELEASE CANDIDATES OR ALPHA RELEASES" 
```

Comment:
I would go one step further and prefix these lines with a "- "  so you can't blindly copy and paste it in.  But I'm ok without that too.